docs/site/Deploying_to_ibm_cloud_kubernetes.md

```diff
@@ -0,0 +1,707 @@
+You should see:
+
+````
+Logging in to 'registry.ng.bluemix.net'...
+Logged in to 'registry.ng.bluemix.net'.
+OK
+````
+
+## Step 14: Upload a docker image to the Container Registry
+
+This requires several steps, let's quickly go through them.
+
+### Create a namespace
+
+List your current namespace by typing:
+
+```
+ibmcloud cr namespace-list
+```
+
+If you want to create a new namespace for yourself, you can do so with this command:
+
+````
+ibmcloud cr namespace-add <my_namespace>
+````
+
+NOTE : I will be using my namespace ````dremond```` in the commands that follow.
+
+
+### Tag your local docker image with the IBM Cloud container registry
+
+Here is the command:
+
+````
+docker tag <source_image>:<tag> registry.<region>.bluemix.net/<my_namespace>/<new_image_repo>:<new_tag>
+````
+
+````<source_image>:<tag>```` is what you have on your machine that you created earlier.
+
+e.g.  lb4-simple-web-app:latest
+
+
+
+````registry.<region>.bluemix.net```` is the container registry region you logged into before.
+
+e.g. registry.ng.bluemix.net
+
+
+````<my_namespace>```` is the namespace you created for yourself
+
+e.g.  dremond   (in my case)
+
+
+````<new_image_repo>:<new_tag>```` can be whatever you want it to be; they don't have to exist yet
+
+e.g.   loopback4_webapp_repo:1
+
+
+So, putting these values together, my command will look like this:
+
+````
+docker tag lb4-simple-web-app:latest registry.ng.bluemix.net/dremond/loopback4_webapp_repo:1
+````
+
+### Push the local image to the container registry
+
+
+````
+docker push registry.ng.bluemix.net/dremond/loopback4_webapp_repo:1
+````
+
+You will see a progress bar like this:
+
+````
+The push refers to repository [registry.ng.bluemix.net/dremond/loopback4_webapp_repo]
+478b1e842aa3: Pushed
+6fd2223ea65e: Pushed
+a90c4aba186a: Pushing [============>                                      ]   51.4MB/207.9MB
+bb288a38c607: Pushed
+53981d6ec3d2: Mounted from dremond/loopback4_repo
+b727cac390f6: Mounted fro
+````
+
+Wait until it is completed.
+
+````
+The push refers to repository [registry.ng.bluemix.net/dremond/loopback4_webapp_repo]
+478b1e842aa3: Pushed
+6fd2223ea65e: Pushed
+a90c4aba186a: Pushed
+bb288a38c607: Pushed
+53981d6ec3d2: Mounted from dremond/loopback4_repo
+b727cac390f6: Mounted from dremond/loopback4_repo
+df64d3292fd6: Mounted from dremond/loopback4_repo
+1: digest: sha256:939ada9d1b7f6a7483aed69dff5ccc28d1931ed249b38d51d34b854b32139177 size: 1787
+````
+
+### Verify the image is in the container registry
+
+Type the command :
+
+````
+ibmcloud cr image-list
+````
+
+You should see your image listed.
+
+![](./imgs/lb4_k8s_ibm_cloud_container_registry_1.png)
+
+
+NOTE: One important column is the last one on the right "Security Status". My image has a "No Issues" status. This is good. If it instead stated "9 Issues" (in red), then I would need to take some extra steps later on in my account to view the issues, figure out a way to resolve them, or create an Exemption for them.
```

Comment:
Please use "note" or "impotant" alert here.

See https://loopback.io/doc/en/contrib/alerts.html#types-of-alerts

Reply:
done